api/src/paths/project/{projectId}/attachments/{attachmentId}/delete.test.ts

```diff
@@ -83,128 +84,15 @@ describe('deleteAttachment', () => {
     }
   });
 
-  it('should throw a 400 error when no sql statement returned for unsecureAttachmentRecordSQL', async () => {
```

Comment:
same question here .... not needed?

Reply:
Tests for unsecure is not needed as we no more switch between secure and unsecure. Also other delete tests are now located in attachment-service.